src/network-interface.js

```diff
@@ -14,12 +17,12 @@ export class HTTPUploadNetworkInterface extends HTTPFetchNetworkInterface {
         operation.query = printAST(operation.query)
 
         // Build the form
-        const formData = new window.FormData()
+        const formData = new FormData()
```

Comment:
Are there linting errors now that `window` is removed everywhere? ESLint config will need to be updated if this is a necessary change.